lsec2.py

```diff
@@ -4,18 +4,23 @@
 
 '''
 USAGE: lsec2 [filter]
-    prints all instances (optionally filtered by name by the `filter` param) and their id, Name, instance type, and DNS.
+    prints all instances (optionally filtered by name by the `filter` param) 
+    and their id, Name, instance type, and DNS.
 '''
 
-def get_ec2_instances(region, name_filter=""):
+def _get_ec2_instances(region, name_filter=""):
     ec2_conn = boto.ec2.connect_to_region(region)
     instances = ec2_conn.get_all_instances()
     for instance in instances:
         i = instance.instances[0]
         if name_filter in i.tags.get('Name', ''):
-            print "{:<10} {:<40} {:<10} {:<20}".format(i.id, i.tags.get('Name', ''), i.instance_type,
-                                                       i.dns_name or "No DNS name")
+            print "{:<10} {:<40} {:<10} {:<20}".format(
```

Comment:
If you're going to put stuff on new lines, you should put the first param on the first line (ie immediately after the open brace), then align the others under it.